lib/rabbit/command/rabbit.rb

```diff
@@ -652,7 +648,7 @@ def choose_source_file_by_dialog
         add_source_dialog_filter(dialog, "Markdown files", "*.md")
         add_source_dialog_filter(dialog, "All files", "*")
         file_name = nil
-        if dialog.run == Gtk::Dialog::RESPONSE_ACCEPT
+        if dialog.run == :accept
```

Comment:
We can't use `:accept` here.
We need to use `Gtk::ResponseType::ACCEPT`.

Reply:
Thank you. I will correct it.